text/src/autogluon/text/automm/data/mixup.py

```diff
@@ -0,0 +1,16 @@
+from timm.data.mixup import Mixup
+
+class MixupModule(Mixup):
```

Comment:
If we use mixup inside a model's forward, e.g., `timm_image.py`, we don't need this wrapper since the keys are not necessary.

Reply:
I have the same question. Why do we need this class?